opentui: add toast & port flags (-c, -s)

veracioux opened the pull request:
- Implemented --continue and --session options
- Closes #3372.
- Created Toast component
<img width="2843" height="706" alt="image" src="https://github.com/user-attachments/assets/f7fc8704-bf63-4bea-8e6d-eba676fd29f6" />
<img width="2828" height="489" alt="image" src="https://github.com/user-attachments/assets/963199c0-a685-4ee6-ae2a-3e96a4f18553" />

veracioux committed:
feat: Port CLI options to opentui version

veracioux committed:
chore: Mark as completed

veracioux committed:
feat: Create Toast component

veracioux committed:
fix: Implement /share copy to clipboard (#3372)

rekram1-node commented:
generally if possible I'd recommend splitting into 2 prs if we can help it

rekram1-node commented on packages/opencode/src/cli/cmd/tui/routes/session/index.tsx:
```diff
@@ -173,12 +176,20 @@ export function Session() {
-          },
-        })
+      onSelect: async (dialog) => {
+        try {
```
can we use .catch instead of try catch

veracioux replied:
Sure thing

veracioux replied:
Done

rekram1-node commented on packages/opencode/src/cli/cmd/tui.ts:
```diff
@@ -73,32 +73,36 @@ export const TuiCommand = cmd({
-          }
-          return undefined
-        })()
+        const sessionID = "make-typechecker-happy"
```
I think you should just leave this file as it was we aren't using it anywhere right? so no reason to make edits unless u wanted to delete it?

veracioux replied:
I wanted to mark stuff that was done. Since there is no such tracking by the other devs rn, I'll remove it before marking this PR as ready. But I'll keep it for now for my own organization.

veracioux committed:
refactor: Replace try-catch with then-catch

rekram1-node commented:
Yeah I think this is good, if you drop that one file edit I think this is good to merge

veracioux committed:
fix: Handle invalid sessionID errors

veracioux committed:
feat: Standardize toast duration to 5s

veracioux committed:
feat: Change session not found error toast to match dev

veracioux commented:
@rekram1-node Please check the newest commits. Now it's ready to merge. Made these changes:
- Changed the toast duration to optional with 5s default to match the dev branch
- Added error handling for invalid sessionID specified in `--session`, matching dev branch
<img width="1317" height="768" alt="image" src="https://github.com/user-attachments/assets/3fb50ab2-c60b-4cf6-a134-a3846ecb2b6c" />

veracioux committed:
Revert "chore: Mark as completed"

This reverts commit 98d6530c092aeeeebf9344b58861aadeb0ae474e.

rekram1-node commented:
I think this is awesome, great work
